admin-dev/themes/new-theme/js/theme.js

```diff
@@ -62,6 +63,13 @@ new NavBar();
 new Header();
 
 $(() => {
+  // Initialize the prestashop global object
+  window.prestashop = {
```

Comment:
`prestashop` is already initialized watch out.

Reply:
BO doesn't have a prestashop object